WebRoot/js/zimbraMail/share/view/dialog/ZmPasswordRecoveryDialog.js

```diff
@@ -965,6 +982,13 @@ function (result) {
 			}
 		}
 
+		if(zimbraPasswordAllowUsername) {
```

Comment:
Is this condition check correct?

Reply:
This condition was changed to check the correct value